tests/fixtures/virt/virtual_machines.py

```
@@ -0,0 +1,150 @@
+    with (
+        instance_type_for_test_scope_class as vm_instance_type,
+        vm_preference_for_test as vm_preference,
+    ):
+        with VirtualMachineForTests(
+            client=unprivileged_client,
+            name="rhel-vm-with-instance-type",
+            namespace=namespace.name,
+            image=Images.Rhel.RHEL9_REGISTRY_GUEST_IMG,
+            vm_instance_type=vm_instance_type,
+            vm_preference=vm_preference,
+        ) as vm:
```

Ruff (0.15.21)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)